File: php/delete.php
<?php
$m = new MongoClient();
echo "Connection to database successfully";
$db = $m -> MYDB259743,
echo "\n Database selected successfully";
$col = $db-> MyCol;
echo "\n Collection selected successfully";
$col->remove(array("name" => "Chanchal"));
echo "\n Document deleted successfully";
?>

File: python/delete.py
from pymongo import MongoClient
client=MongoClient('localhost',27017)
db=client.TYITDB259743
def delete():
    try:
        name1=input("Enter the Name: ")
        db.MyCol.delete_one({"name":name1})
        print("\nData deleted successfully")
    except Exception:
        print(str(e))

delete()
